client/src/components/Sidebar/modules/MainMenu.tsx

```diff
@@ -202,21 +203,38 @@ export const Menu: React.FunctionComponent<MenuProps> = ({
         }
       >
         <button
-          className="sidebar-footer__account"
+          className="
+          sidebar-footer__account
+          w-bg-primary
+          w-text-white
+          w-flex
+          w-items-center
+          w-relative
+          w-p-0
+          w-w-full
+          w-appearance-none
+          w-border-0
+          w-overflow-hidden
+          md:w-px-5
+          md:w-py-3
+          hover:w-bg-primary
+          hover:w-text-white focus:w-bg-primary focus:w-text-white"
           title={strings.EDIT_YOUR_ACCOUNT}
           onClick={onClickAccountSettings}
           aria-label={strings.EDIT_YOUR_ACCOUNT}
           aria-haspopup="menu"
           aria-expanded={accountSettingsOpen ? 'true' : 'false'}
           type="button"
         >
-          <div className="avatar square avatar-on-dark">
+          <div className="avatar avatar-on-dark !w-w-[28px] !w-h-[28px]">
             <img src={user.avatarUrl} alt="" />
           </div>
           <div className="sidebar-footer__account-toggle">
-            <div className="sidebar-footer__account-label">{user.name}</div>
+            <div className="sidebar-footer__account-label w-label-3 w-text-white">
+              {user.name}
+            </div>
             <Icon
-              className="sidebar-footer__account-icon"
+              className="w-w-4 w-h-4 w-text-white"
```

Comment:
Considering it seems the text is white for almost everything in here, if we’re refactoring the code, I’d prefer we set `w-text-white` on the whole sidebar and then use `inherit` in places like this.